pages/linux/expect.md

```diff
@@ -0,0 +1,16 @@
+# expect
+
+> Programmed dialogue with interactive programs.
```

Comment:
Hrm, this is an awkward one because this is what the man page says, but it's a bit on the confusing side. Would something like this still accurately reflect what the command does, @einverne?
```suggestion
> Script executor that interacts with other programs that require user input.
```